platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -140,6 +142,13 @@ public void init() throws Exception {
         return export(Paths.get(exportDirectory));
     }
 
+    @Override
+    public Collection<Describable> getInfoForOptionalMigratables() {
+        return dataMigratables.stream()
```

Comment:
Why are we using a stream here to copy the list? We should be able to create a new list and do  `Collections.addAll(dataMigratablesCopy, dataMigratables)`, or use guava's `ImmutableList.copyOf(dataMigratables)`